Modules/Backup-VCSA/Backup-VCSA.psm1

```diff
@@ -1,204 +1,462 @@
-	
+				$backups[0] | Get-VCSABackupStatus
 	.NOTES
 		The BackupID can be piped in from the Get-VCSABackupJobs function and can return multiple job statuses
 #>
-    Param (
-        [parameter(ValueFromPipeline=$True)]
-        [string[]]$BackupID
-    )
- Begin {
-        if (!($global:DefaultCisServers)){ 
-            [System.Windows.Forms.MessageBox]::Show("It appears you have not created a connection to the CisServer. You will now be prompted to enter your vCenter credentials to continue" , "Connect to CisServer") | out-null
-            $Connection = Connect-CisServer $global:DefaultVIServer 
-        } else {
-            $Connection = $global:DefaultCisServers
-        }
-        
-        $BackupAPI = Get-CisService com.vmware.appliance.recovery.backup.job
- }
-    Process{
-       
-        foreach ($id in $BackupID) {
-            $BackupAPI.get("$id") | select id, progress, state
-        }
-        
+	Param (
+		[parameter(ValueFromPipeline=$True)]
+		[string[]]$BackupID
+	)
+	Begin {
+		if (!($global:DefaultCisServers)){ 
+			[System.Windows.Forms.MessageBox]::Show("It appears you have not created a connection to the CisServer. You will now be prompted to enter your vCenter credentials to continue" , "Connect to CisServer") | out-null
+			$Connection = Connect-CisServer $global:DefaultVIServer 
+		} else {
+			$Connection = $global:DefaultCisServers
+		}
+		$BackupAPI = Get-CisService com.vmware.appliance.recovery.backup.job
+	}
+	Process{
+		foreach ($id in $BackupID) {
+			$BackupAPI.get("$id") | select id, progress, state
+		}
+	}
+	End {}
+}
+
+Function Create-VCSASchedule {
+<#
+	.NOTES
+	===========================================================================
+	 Original Created by:  Brian Graf
+	 Blog:                 www.vtagion.com
+	 Twitter:              @vBrianGraf
+	 Organization:         VMware
+	 Created / Modifed by: Michael Dunsdon
+	 Twitter:              @MJDunsdon
+	 Date:                 September 16, 2020
+	===========================================================================
+
+	.SYNOPSIS
+		This function will allow you to create a scheduled to backup your
+		VCSA appliance. (vSphere 6.7 and higher)
+	.DESCRIPTION
+		Use this function to create a schedule to backup your VCSA to a remote location
+	.EXAMPLE
+		The Below Create a schedule on Monday @11:30pm to FTP location 10.1.1.10:/vcsabackup/vcenter01
+		and keep 4 backups with a Encryption Passowrd of "VMw@re123"
+		
+		$location = "ftp://10.1.1.10/vcsabackup/vcenter01"
+		$LocationUser = "admin"
+		[VMware.VimAutomation.Cis.Core.Types.V1.Secret]$locationPassword = "VMw@re123"
+		$BHour = 23
+		$BMin = 30
+		$BDays = @("Monday")
+		$MaxCount = 4
+		[VMware.VimAutomation.Cis.Core.Types.V1.Secret]$BackupPassword = "VMw@re123"
+		
+		PS C:\> Set-VCSASchedule -Location $location -LocationUser $LocationUser -LocationPassword $locationPassword -BackupHour $BHour -BackupMinute $BMin -backupDays $BDays -MaxCount $MaxCount -BackupPassword $BackupPassword
```

Comment:
Function name is wrong here. It should be Create-VCSASchedule and after applying rename suggestion it should be New-VCSASchedule